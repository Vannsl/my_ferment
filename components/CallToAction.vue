
<template>
  <div>
    <call-to-action-wave-svg />
    <call-to-action-content />
  </div>
</template>

<script>
import CallToActionContent from '@/components/CallToActionContent'
import CallToActionWaveSvg from '@/components/svg/CallToActionWave'

export default {
  name: 'CallToAction',
  components: {
    'call-to-action-content': CallToActionContent,
    'call-to-action-wave-svg': CallToActionWaveSvg
  }
}
</script>