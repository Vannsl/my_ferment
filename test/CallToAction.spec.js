import { shallowMount } from '@vue/test-utils'
import CallToAction from '@/components/CallToAction'

const factory = () => {
  return shallowMount(CallToAction)
}

describe('CallToAction', () => {
  it('is a Vue instance', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
