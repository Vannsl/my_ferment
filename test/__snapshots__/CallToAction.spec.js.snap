// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CallToAction renders properly 1`] = `"<div><call-to-action-wave-svg-stub></call-to-action-wave-svg-stub> <call-to-action-content-stub></call-to-action-content-stub></div>"`;
